adhocracy4/maps/widgets.py

```diff
@@ -71,14 +75,23 @@ def render(self, name, value, attrs, renderer=None):
         if hasattr(settings, "A4_OPENMAPTILES_TOKEN"):
             omt_token = settings.A4_OPENMAPTILES_TOKEN
 
+        point = value
```

Comment:
why not just replace `value` with `point` above where the method is defined?

Reply:
`value` comes from the super method, won't this break it if we change the name here?

Reply:
but you are replacing the super method, and it's just a positional argument

Reply:
hmm, I just tested it and I did get an error: it's actually used as a named argument here: https://github.com/django/django/blob/main/django/forms/boundfield.py#L109